tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
```

Comment:
As mentioned in above comment, we should use `go fmt` instead of `goimports`. Thus we don't need this.

Reply:
I disaggree as the imports will not follow a standardized sorting check.
I started very simple with this CI branch, but eventually I would go with something like `golangci`, which is used by many OSS projects.

here an example from the helm repo.

https://github.com/helm/helm/blob/main/.golangci.yml